docs/high-availability/kube-controller-manager.manifest

```diff
@@ -11,7 +11,7 @@
     "command": [
                  "/bin/sh",
                  "-c",
-                 "/usr/local/bin/kube-controller-manager --master=127.0.0.1:8080 --cluster_name=e2e-test-bburns --cluster-cidr=10.245.0.0/16 --allocate-node-cidrs=true --cloud_provider=gce  --service_account_private_key_file=/srv/kubernetes/server.key --v=2 1>>/var/log/kube-controller-manager.log 2>&1"
+                 "/usr/local/bin/kube-controller-manager --master=192.168.4.102:8080 --service_account_private_key_file=/srv/kubernetes/server.key --v=4  1>>/var/log/kube-controller-manager.log 2>&1"
```

Comment:
do you mean to say "this will only work if kube2 is up", if so i agree.  I made it a static IP just because the 127.0.0.1 implies you need apiserver running on every node, when in reality all you need is a load balanced endpoint.  

Im not against using 127.0.0.1, but just thought this would be representative of a more real-world scenario where you dont need api-servers running on every possible controller manager host.